### DISCUSSION TITLE
Linkコンポーネントで「/」に遷移したい

### DISCUSSION THREAD

**Discussion by lainNao** [uhyo/rocon#37] [Q&A] [answered]
[ドキュメント](https://rocon.uhyohyo.net/docs/component/link)にあるサンプルでは「/foo」や「/bar」に対するリンクの作り方については以下のようにやればよいと記載が見つかりました。
```tsx
<Link route={toplevelRoutes._.fooかbar}>... 
```
しかし「/」に対して遷移させる場合の例を見つけることができませんでした。
以下のような「I am top page」のページに遷移するにはどう書けばよいでしょうか？

```ts
const toplevelRoutes = Rocon.Path()
  .exact({
    action: () => <p>I am top page</p>      // <Link route={????}> ...
  })

```

**Comment by uhyo** [comment, marked as answer]
例が不足しており申し訳ありません 🥺 その場合は `toplevelRoutes.exactRoute` で遷移できます。

**Reply by lainNao** [reply]
@uhyo [こちら](https://rocon.uhyohyo.net/docs/builder/path)のexactRouteがそちらだったのですね、ありがとうございます！！
